aws/tailers/stack_event.go

```diff
@@ -0,0 +1,229 @@
+package awstailers
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	"text/tabwriter"
+
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/fatih/color"
+	"github.com/wallix/awless/aws/services"
+)
+
+type stackEventTailer struct {
+	stackName        string
+	refresh          bool
+	watchDeployment  bool
+	refreshFrequency time.Duration
+	lastEventID      *string
+	nbEvents         int
+}
+
+type stackEvents []*cloudformation.StackEvent
+
+func coloredResourceStatus(str string) string {
```

Comment:
A detail but, helpers/boilerplates code should go at the end of the file. We should prefer strong public/business logic at the top of the file.